css/index.css

```diff
@@ -136,11 +206,139 @@ footer nav {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    padding: 2% 2%;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media (max-width: 768px) {
```

Comment:
There is a horizontal scroll bar when I go to tablet mode! Oh no! But other than that, your view for the tablet is very close to the design files that were provided.